.github/workflows/ci-1.85.yml

```diff
@@ -0,0 +1,39 @@
+# This job checks that picus and the halo2 frontend without the llzk deps builds for 1.85
+
+name: Continuous Integration (1.85)
```

Comment:
What is 1.85 associated with?

Reply:
Cool. Maybe we could reference `rust-1.85` instead of just `1.85` to clarify, but that's minor.